bin/concatenate_all.py

```diff
@@ -103,6 +102,35 @@ def main():
             print("ERROR! ", dir, " is not a valid input directory for Lichen")
             exit(1)
 
+
+def main():
+    start_time = time.time()
+    args = parse_args()
+
+    sys.stdout.write("CONCATENATE ALL...")  # don't want a newline here so can't use print
+    sys.stdout.flush()
```

Comment:
You can use the named parameter `end` in the [`print`](https://docs.python.org/3/library/functions.html#print) function to disable the newline.

```suggestion
    print("CONCATENATE ALL...", end="")
```